app/components/Form/SubmitBtns.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+import Button from '@button-inc/bcgov-theme/Button';
+import LoadingSpinner from 'components/LoadingSpinner';
+
+const StyledFlex = styled('div')`
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width: 600px) {
+    flex-direction: row;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  margin: 24px 0;
+  white-space: nowrap;
+  min-width: 178px;
+
+  @media (min-width: 600px) {
+    margin: 0 24px;
+  }
+`;
+
+const StyledToast = styled('div')`
+  background-color: ${(props) => props.theme.color.successGreen};
+  border-radius: 4px;
+  padding: 12px 24px;
+  color: #ffffff;
+  font-size: 16px;
+  transition: all ease-in-out 0.2s 0.2s;
+  text-align: center;
+
+  & a {
+    color: #ffffff;
+  }
+`;
+
+type Props = {
+  disabled: boolean;
+  formData: any;
+  isSubmitPage: boolean;
+  isUpdating: boolean;
+  saveAsDraft: boolean;
+  saveForm: any;
+  status: string;
+};
+
+const SubmitBtns = ({
+  disabled,
+  formData,
+  isSubmitPage,
+  isUpdating,
+  saveAsDraft,
+  saveForm,
+  status,
+}: Props) => {
+  const isWithdrawn = status === 'withdrawn';
+  const isDraft = status === 'draft';
+
+  const formatSubmitBtn = () => {
+    if (isWithdrawn) {
+      return 'Continue';
+    }
+    if (!isSubmitPage) {
+      return 'Save and continue';
+    }
+    return 'Submit';
+  };
+
+  return (
+    <StyledFlex>
+      <Button variant="primary" disabled={disabled}>
+        {formatSubmitBtn()}
+      </Button>
+      {isSubmitPage && isDraft && (
+        <>
+          <StyledButton
+            variant="secondary"
+            onClick={(e: React.MouseEvent<HTMLInputElement>) => {
```

Comment:
Thoughts on passing a `handleSubmit` function rather than just the saveForm function?

Reply:
I believe there is a few reasons for this.  We are using ``e.preventDefault()` because RJSF will use any child button as a submit button if not. The `handleSubmit` function also will change the form page once completed.   
  
Also you can see I added an argument `isSaveAsDraftBtn` to the `saveForm` function which was needed to set the state for the save button once the mutation had completed.

Reply:
Ah sorry I meant moreso to send a function that fully handles the `onClick` in this case, but this is more a matter of my own style.